src/rational.rs

```diff
@@ -113,6 +130,14 @@ impl Rational {
     }
 
     /**
+     * Returns this Rational to the nearest Int
+     */
+    pub fn round(&self) -> Int {
+        let r = self + Rational::new(self.sign().into(), 2.into());
```

Comment:
Oh right. Didn't think of that. Hmm... Let me think for a bit. I don't really want to allocate for what is essentially a constant.